lib/node_modules/@stdlib/blas/ext/base/sasumpw/include/stdlib/blas/ext/base/sasumpw.h

```diff
@@ -1,7 +1,7 @@
 /**
 * @license Apache-2.0
 *
-* Copyright (c) 2020 The Stdlib Authors.
+* Copyright (c) 2024 The Stdlib Authors.
```

Comment:
```suggestion
* Copyright (c) 2020 The Stdlib Authors.
```

This change is not necessary.